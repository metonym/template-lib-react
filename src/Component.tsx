import * as React from 'react';

function Component() {
  const [data] = React.useState([0, 1, 2]);

  return (
    <>
      {data.map(item => (
        <div key={item}>{item}</div>
      ))}
    </>
  );
}

export default Component;
