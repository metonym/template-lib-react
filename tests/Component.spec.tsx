import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as React from 'react';
import Component from '../src/Component';

describe('Component', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<Component />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
