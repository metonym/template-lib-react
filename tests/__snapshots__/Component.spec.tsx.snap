// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component matches the snapshot 1`] = `
<Fragment>
  <div
    key="0"
  >
    0
  </div>
  <div
    key="1"
  >
    1
  </div>
  <div
    key="2"
  >
    2
  </div>
</Fragment>
`;
